dbt/adapters/databricks/connections.py

```diff
@@ -1120,21 +1121,31 @@ def _get_compute_name(node: Optional[ResultNode]) -> Optional[str]:
 
 
 def _get_http_path(node: Optional[ResultNode], creds: DatabricksCredentials) -> Optional[str]:
+    thread_id = (os.getpid(), get_ident())
     # Get the http path of the compute resource specified in the node's config.
     # If none is specified return the default path from creds.
     compute_name = _get_compute_name(node)
     if not node or not compute_name:
```

Comment:
This structure has become a little weird.  I believe we can drop the `if not node`.

Reply:
maybe we can't due to the creds.http_path.  Still, worth thinking about restructuring because its so odd to check if something is not, and then immediately check if it is.

Reply:
Given this weirdness, let's add either a unit test or an end to end test that validates we do the expected thing in the default compute case.

Reply:
I agree that `if not node or not compute_name:` is an odd structure.  However I originally did it that way because using just `if not compute_name:` produced linting errors.  I'll re-organize things.

Reply:
I think the existing unit tests cover all the possibilities for what can be passed in to _get_http_path